doc/RELEASE_NOTES.md

```diff
@@ -0,0 +1,828 @@
+
+_**NOTE WELL**_: like all the other "tools" the `pbench-agent` supports, the `node-exporter` and `dcgm` tools themselves need to be installed separately on the registered hosts.
+
+The new `dcgm` tool requires Python 2, an Nvidia based install which might conflict with the Pbench Agent's Python 3 operational requirement in some cases.
+
+### The PCP tool
+
+Just like the new Prometheus based tools, you can register "PCP" as a persistent tool using: `pbench-register-tool --name=pcp`.  This will cause each Tool Meister on the hosts for which PCP is registered to start a PMCD instance, and the Tool Data Sink will run `pmlogger` processes for each of hosts to collect the data at the requested interval.
+
+The PCP support also allows you to register PCP as a transient tool, where it is started and stopped around each benchmark invoked.  Use the name `pcp-transient` when registering (e.g. `pbench-register-tool --name=pcp-transient`).
+
+_**NOTE AS WELL**_: like all the other "tools" the `pbench-agent` supports, the `pcp` tools themselves need to be installed separately on the registered hosts.
+
+
+## CLI Interfaces to the Tool Meister Sub-system
+
+Two new CLI command are provided to start and stop the Tool Meister sub-system, `pbench-tool-meister-start` and `pbench-tool-meister-stop`.  These two interfaces are mostly used by the benchmark convenience scripts provided by the Pbench Agent.
+
+If you were using the `pbench-start-tools`, `pbench-stop-tools`, and `pbench-postprocess-tools` before, you must now invoke `pbench-tool-meister-start` before using the start/stop/postprocess tool interfaces.
+
+Further the `pbench-send-tools` CLI interface has been added to instruct the Tool Meister sub-system when it is time to send the collected data from transient tools to the Tool Data Sink.  The typical sequence is to "start", "stop", "send", "post-process".  Now you can "send" and "post-process" the transient tool data any time before `pbench-tool-meister-stop` is called.
+
+
+## Independence of Pbench Agent "tool" Scripts
+
+The tool scripts the Pbench Agent uses to collect data can be run independent of the rest of the Pbench Agent so that users can verify they collect data as expected.
+
+
+## Default Tool Set Reduced to `iostat`, `sar`, & `vmstat` Tools
+
+The default tool set registered using `pbench-register-tool-set` is now only three tools: `iostat`, `sar`, and `vmstat`.  Prior to this release the default tool set included some very heavy-weight tools like `pidstat`, `proc-interrupts`, and `perf` (`perf record`).
+
+The default tool set from v0.69 is preserved in a new tool set named, `legacy`, and can be loaded via `pbench-register-tool-set --toolset=legacy`.
+
+Along with this change, 3 new named tool sets have also be added:
+
+ * `light`: `vmstat`
+ * `medium`: ${light}, `iostat`, `sar` (this is the new default tool set)
+ * `heavy`: ${medium}, `perf`, `pidstat`, `proc-interrupts`, `proc-vmstat`, `turbostat`
```

Comment:
It's probably worth having a trailing comment that users aren't required to use the pre-defined sets, that they can register whichever tools they like.